__manifest__.py

```diff
@@ -11,7 +11,8 @@
     "data": [
         "views/openacademy_menu.xml",
         "views/openacademy_course_views.xml",
-        "views/openacademy_session_views.xml"
+        "views/openacademy_session_views.xml",
+        "views/openacademy_res_partner_views.xml"
```

Comment:
Pro tip: if you always provide a trailing comma, you don't need to modify the last line of a list or dictionary every time you want to add a new line, so the diff would be smaller.

On the other hand, it should be the model name:

```suggestion
        "views/res_partner_views.xml",
```